src/services/API/User.js

```diff
@@ -3,23 +3,28 @@ import {
   otpRequestEndpoint,
   otpVerifyEndpoint,
   userFromTokenEndpoint,
+  usersEndpoint,
+  userConfigEndpoint,
 } from "@/services/API/Endpoints.js";
 
 export default {
   requestOtp(mobile) {
+    // request OTP for login
     return apiClient().post(process.env.VUE_APP_BACKEND + otpRequestEndpoint, {
```

Comment:
`process.env.VUE_APP_BACKEND` is not needed right? It's already being added a the baseUrl in RootClient.js